net.lutris.Lutris.yml

```diff
@@ -59,7 +59,7 @@ add-extensions:
     merge-dirs: bin
 
 sdk-extensions:
-  - org.freedesktop.Sdk.Compat.i386
+  - org.gnome.Sdk.Compat.i386
   - org.freedesktop.Sdk.Extension.toolchain-i386
```

Comment:
@abderrahim AFAIK the `toolchain-i386` extension is built against FD.o SDK. Will it keep working on Gnome SDK just as fine?

Reply:
Yes, it should. We inherit all the sdk extensions from freedesktop-sdk.